BREAKING.md

```diff
@@ -24,6 +24,21 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 The `"batchEnd"` listener in `ISummarizerRuntime` has been deprecated and will be removed in a future release. Please remove all usage and implementations of `ISummarizerRuntime.on("batchEnd", ...)` and `ISummarizerRuntime.removeListener("batchEnd", ...)`.
 If these methods are needed, please refer to the `IContainerRuntimeBase` interface.
 
+### Deprecated events and event parameters on IContainer and IDeltaManager
+
+The following legacy events and event parameters have been marked as deprecated:
+
+- IContainerEvents
+    - "contextChanged": Event deprecated in its entirety. No replacement API recommended.
+    - "dirty": Event parameter "dirty" deprecated. The parameter is unneeded, as the event itself signals the current "dirty" state (true).
+    - "saved": Event parameter "dirty" deprecated. The parameter is unneeded, as the event itself signals the current "dirty" state (false).
+- IDeltaManagerEvents
```

Comment:
For "No replacement API ..." please add short description with some reasoning around deprecation.